packages/filesystem/package.json

```diff
@@ -10,7 +10,8 @@
     "@types/tar-fs": "^1.16.1",
     "async-mutex": "^0.3.1",
     "body-parser": "^1.18.3",
-    "browserfs": "^1.4.3",
+    "@zenfs/core": "^0.16.4",
+    "@zenfs/dom": "^0.2.14",
```

Comment:
This change (and the suggested renames) need to be listed as breaking changes in the Changelog